internal/ethapi/api.go

```diff
@@ -800,21 +800,21 @@ func (args *CallArgs) ToMessage(globalGasCap uint64) types.Message {
 	return msg
 }
 
-// account indicates the overriding fields of account during the execution of
+// Account indicates the overriding fields of account during the execution of
 // a message call.
 // Note, state and stateDiff can't be specified at the same time. If state is
 // set, message execution will only use the data in the given state. Otherwise
 // if statDiff is set, all diff will be applied first and then execute the call
 // message.
-type account struct {
+type Account struct {
```

Comment:
This is an  internal packag,e you can't use it as an API surface, Go won't allow outside callers to access it. All in all internal is meant to be used internally by Geth.

Reply:
ahh - that's right - okay so I either move it out or make a same type in ethclient.go - i wonder if a type alias would do the trick here

Reply:
just tried - a type alias worked great - I make an account in outside go-ethereum code just fine